src/SmartComponents/PatchSetWizard/steps/RequestProgress.js

```diff
@@ -44,41 +46,44 @@ const RequestProgress = ({ onClose, state }) => {
             />
             <Title headingLevel="h1" size="lg">
                 {failed
-                    ? 'Something went wrong'
+                    ? intl.formatMessage(messages.textErrorSomethingWrong)
                     : requestPending
-                        ? 'Configuration in progress'
-                        : 'Patch template configuration successful'}
+                        ? intl.formatMessage(messages.textConfigurationInProgress)
+                        : intl.formatMessage(messages.textPatchTemplateSuccessfuly)}
             </Title>
             <EmptyStateBody>
                 <Grid hasGutter>
                     {(requestPending) && (
                         <><GridItem>
                             <HelperText>
                                 <HelperTextItem variant="indeterminate">
-                                    Please allow a few minutes to set up a patch template.
-                                    You will receive a notification when finished</HelperTextItem>
+                                    {intl.formatMessage(messages.textPatchTemplatePending)}
```

Comment:
I didn't run it locally, but did you check whether the newline was rendered here? IIRC, I had a problem with newliners while formatting messages with formatjs

Reply:
The actual expectation is to make it inline :)